src/sshagent/SSHAgent.cpp

```diff
@@ -120,30 +130,37 @@ bool SSHAgent::isAgentRunning() const
 bool SSHAgent::sendMessage(const QByteArray& in, QByteArray& out)
 {
 #ifdef Q_OS_WIN
-    if (!useOpenSSH()) {
```

Comment:
This section needs to be rewritten.

It would probably clean this logic up a lot if the OpenSSH stuff is moved under a new function `sendMessageOpenSSH` which always exists and `sendMessage` only has the selection logic in. You also need to change it so that if either of them return false you return false immediately without calling the other so the error is handled properly by the caller.

You can also set an error and return false here if both agents are disabled as it's a configuration error and would otherwise be a silent fail.

Additionally run the `format` target of CMake to make sure the code adheres to the formatting standards we have.